src/popup/components/AccountSelector.vue

```diff
@@ -1,13 +1,14 @@
 <template>
   <div class="account-selector">
-    <Avatar :address="value" />
+    <Avatar :address="modelValue.toString()" />
     <div>
       <BtnPill
         class="account-select"
         dense
       >
         <FormSelect
-          :value="value"
+          v-bind="$attrs"
+          :model-value="modelValue"
```

Comment:
Here the `modelValue` is not being propagated further to the parent.
`@update:modelValue="$emit('update:modelValue', $event)"` adding this line will fix it. Not sure that it is the best way, but fastest.

Reply:
Great catch! Thank you! Usually we would use `v-model` instead of `:model-value` but `eslint` complains that we are _mutating the "modelValue" prop_ 
Propagating the `modelValue` like you suggest is exactly what `v-model` does as well, only that the linter is not complaining so I believe this is a good way to fix this